### DISCUSSION TITLE
Topics for potential research:  communication, collaboration, self-overcoming, overcoming structure agency, transcendence

### DISCUSSION THREAD

**Discussion by martinprogrammer** [life-itself/community#1149] [General]
Something that interests me - feel free to respond by adding your interests and ideas. 

1) The absolute limits of human "being-in-the world", "being with others", relationships and collaboration.  
2) Transcendence through self-overcoming and the overcoming of structure agency.  
3) Theory of change
4) Systems theory and systems corruption
5) Power dynamics

This is a mixed bag of philosophers and their thoughts offered here for consideration. 

**Heidegger** - Mitsein (being-with) - proposes that humans exist in relation with each other and that communication is an essential way of relating shared existence.  Transcendence is projecting possibilities into the future through their projects and aspirations.

We are characterised by our openness to possibilities!!!  We have agency in terms of how we go about realising the possibilities and being open to them is the first step.

Our attunement (or mood) is one of the aspect of how we participate in the world.  We reveal ourselves through the mood and our reality is directly influenced by our state-of-mind (or mood).  Not only that we reveal us to the world with our mood - but the world reveals itself to us through the same prism.

**Buber** - says that the transcendence is experienced through relationships with others and with God.  Key to overcoming limitations of one's existence is through other!

**Marcel** - bases his philosophy on the importance of personal relationships and commitment to each other and touches on the sacredness of individual experience.

**Arendt** - the definition of power as the ability to act in concert.  Advocating for public discourse to collectively build power for change.  Freedom is reached through plurality, power and creating of a common vision for the world.

**Nietzsche** - a lot of interesting thoughts about dynamics of power and individuality.  Will to power,  the notion of ubermensch emphasising the importance of authenticity, self expression, self overcoming and going beyond structure agency.

**de Beauvoir** - individual freedom and authenticity - going beyond structure agency.  Love, power dynamics and gender dynamics.

**Deleuze** - Celebrating becoming over being.  Difference is the defining characteristic not the sameness.

**Bergson** - argued that the intuition rather than intellect is the best tool for sense making.  His concept of "elan vital" is similar to will to power.  Creativity is the process of breaking structure agency and tapping into vital energy.  Freedom is similar to Nietzsche's authenticity and making rules rather than adopting them.

**Sartre** - we need to define our own essence and act accordingly.  Live authentically and be free through making our own choices.

**Comment by JonahWilberg** [comment]
Thanks Martin, my background is in philosophy, and I totally agree that the history of philosophy offers a rich resource for understanding the Second Renaissance. I've therefore included Philosophy among the core disciplines that I think feed into 2R reseach [here](https://wiki.secondrenaissance.net/wiki/Potential_research_agenda). I would add particularly: Hegel, Marcuse, Zizek, Badiou, and also ancient philosophy - which was of course in part what inspired the First Renaissance!

**Comment by rufuspollock** [comment]
@martinprogrammer i think this is a nice list and I would be interested in more detail about relation to our overall research context.

Also it would be useful to go through the archives here (are you up for that?) and collecting existing materials e.g. i found this quickly ...

- Research and overview: How do paradigms arise in society and get adopted? - https://github.com/orgs/life-itself/discussions/72
- And especially this bigger earlier thread https://github.com/orgs/life-itself/discussions/591

I think there may be more!

**Reply by martinprogrammer** [reply]
> @martinprogrammer i think this is a nice list and I would be interested in more detail about relation to our overall research context.
> 
> Also it would be useful to go through the archives here (are you up for that?) and collecting existing materials e.g. i found this quickly ...
> 
> * Research and overview: How do paradigms arise in society and get adopted? - https://github.com/orgs/life-itself/discussions/72
> * And especially this bigger earlier thread https://github.com/orgs/life-itself/discussions/591
> 
> I think there may be more!

My primary interest and the first step is analysing existing limitations in communication, relationships, collaboration, processes and structures that hinder our research collective potential. This sets the stage for the need for change and optimisation.  This seems like the logical next step.  

One of the frustrations in particular is the asynchronous communication which leads to a lot of loose ends.

1) Delayed responses
2) Lack of feedback
3) High volume of topics that lead to information overload
4) Lack of conclusions / closure

**Reply by catherinet1** [reply]
resonate particularly with "lack of conclusions/closure". thanks for articulating these frustrations, @martinprogrammer.
I can imagine finding the organising framework of SCQH and issue trees to track research efforts and how individual outputs contribute to overarching questions as proposed by @rufuspollock in https://docs.google.com/document/d/1nr-vnNcCBT5z_rEX7PAft45aCrHvIprwb1F7RkBUHGg/edit?tab=t.0 satisfying and helpful in relation to this point.

**Comment by asimong** [comment]
Thanks to Rufus for re-flagging this!

To go back to your original, Martin, what I would suggest (and what I would
personally like to see) would be that you and all put these lists on your
wiki researcher page. We can create (initially, stub) pages for the authors
in question, and rather than just naming them we can all link to them. A
"what links here" can then reveal who has linked to those people, and you
can contact them with the idea of adding relevant detail to their wiki
pages.

Concerning wiki pages for well-known people: to me, the sensible thing to
do is to start with a link to their wikipedia page. Then focus on specific
features (and linked works) that are relevant to 2R. Then have a
"Commentary" section where all of us can record our personal perspective on
the author.

Best

Simon


On Wed, 11 Dec 2024 at 16:05, martinprogrammer ***@***.***>
wrote:

> Something that interests me - feel free to respond by adding your
> interests and ideas.
>
>    1. The absolute limits of human "being-in-the world", "being with
>    others", relationships and collaboration.
>    2. Transcendence through self-overcoming and the overcoming of
>    structure agency.
>    3. Theory of change
>    4. Systems theory and systems corruption
>    5. Power dynamics
>
> This is a mixed bag of philosophers and their thoughts offered here for
> consideration.
>
> *Heidegger* - Mitsein (being-with) - proposes that humans exist in
> relation with each other and that communication is an essential way of
> relating shared existence. Transcendence is projecting possibilities into
> the future through their projects and aspirations.
>
> We are characterised by our openness to possibilities!!! We have agency in
> terms of how we go about realising the possibilities and being open to them
> is the first step.
>
> Our attunement (or mood) is one of the aspect of how we participate in the
> world. We reveal ourselves through the mood and our reality is directly
> influenced by our state-of-mind (or mood). Not only that we reveal us to
> the world with our mood - but the world reveals itself to us through the
> same prism.
>
> *Buber* - says that the transcendence is experienced through
> relationships with others and with God. Key to overcoming limitations of
> one's existence is through other!
>
> *Marcel* - bases his philosophy on the importance of personal
> relationships and commitment to each other and touches on the sacredness of
> individual experience.
>
> *Arendt* - the definition of power as the ability to act in concert.
> Advocating for public discourse to collectively build power for change.
> Freedom is reached through plurality, power and creating of a common vision
> for the world.
>
> *Nietzsche* - a lot of interesting thoughts about dynamics of power and
> individuality. Will to power, the notion of ubermensch emphasising the
> importance of authenticity, self expression, self overcoming and going
> beyond structure agency.
>
> *de Beauvoir* - individual freedom and authenticity - going beyond
> structure agency. Love, power dynamics and gender dynamics.
>
> *Deleuze* - Celebrating becoming over being. Difference is the defining
> characteristic not the sameness.
>
> *Bergson* - argued that the intuition rather than intellect is the best
> tool for sense making. His concept of "elan vital" is similar to will to
> power. Creativity is the process of breaking structure agency and tapping
> into vital energy. Freedom is similar to Nietzsche's authenticity and
> making rules rather than adopting them.
>
> *Sartre* - we need to define our own essence and act accordingly. Live
> authentically and be free through making our own choices.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/life-itself/discussions/1149>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAURDB5X7SGNJLJ6TQ5J3Y32FBPDNAVCNFSM6AAAAABTN2KO7CVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGY2TQOJXGM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
from Simon Grant +32 485821368 (+44 7710031657 – "asimong" with many
services)

**Reply by asimong** [reply]
I've already said I'm happy to start a fortnightly (or even weekly, if in demand) wiki club, where anyone can come along, either just to be inducted, or to work with others on wiki pages. I've said this before, but I can't say it too often, that to me the ideal is that pages have more than one maintainer, wherever possible. Let's work in relation, collaboratively, from the ground up!

**Comment by asimong** [comment]
Also, going back to your original, @martinprogrammer , could you perhaps convert your initial numbered list into a list of questions that engage you, and could engage others? Like, for instance, the simple label "power dynamics" could cover a multitude of aspects and questions, but if you were to pose a question that is clearly relevant to 2R, that might act as an attractor.

**Comment by asimong** [comment]
What feels vital for a good sense of community is that everyone has an equal chance, and indeed encouragement and support, to bring forward their own potential research topics on an equal footing. **That's** where priorities start to be established, when everyone is heard. Quite similar to a check-in round in any meeting followed by assembling an agenda. Of course, naturally, some items will be seen as more significant, interesting, engaging, urgent, etc. but it's much easier to get buy-in for that when everyone has had a chance to pitch. Another similarity is Open Space Technology.
